tests/testthat/test-sem.R

```diff
@@ -60,7 +60,7 @@ test_that("Multigroup, multimodel SEM works", {
   options$standardizedResidual        = TRUE
   options$pathPlotParameter           = TRUE
   options$standardizedEstimate        = TRUE
-  options$modelTest                   = "satorraBentler"
```

Comment:
hm why does this need to be renamed? I see the QML uses `satorraBentler`: https://github.com/jasp-stats/jaspSem/blob/88a74910962227de43b1441708ad6a3bdba4a50e/inst/qml/SEM.qml#L256